src/components/ChattingRoom/ChatBubble.tsx

```diff
@@ -0,0 +1,76 @@
+import { useRecoilState } from "recoil";
+import styled from "styled-components";
+import chat from "../../interface/chat";
+import { currentId } from "../../store/recoil/recoil";
+import { UserProfileImage } from "../UserProfile";
+import UserData from "../../store/UserData.json";
+
+interface bubbleStyle {
+  color?: string;
+  flexDirection?: string;
+}
+
+function ChatBubble({ chatId, senderId, receiverId, text, time }: chat) {
+  const currentUser = useRecoilState(currentId);
+
+  return (
+    <>
+      {currentUser[0] === senderId ? (
+        <BubbleList justifyContent="flex-end">
+          <TimeStyle>{time}</TimeStyle>
+          <div>
+            <UserName>{UserData.users[senderId - 1].userName}</UserName>
+            <Bubble>{text}</Bubble>
+          </div>
+        </BubbleList>
+      ) : (
+        <BubbleList>
+          <UserProfileImage
+            src={`img/user${senderId}.png`}
+            width={2}
+            height={2}
+          />
+          <div style={{ marginLeft: "0.3rem" }}>
+            <UserName flex="row">
+              {UserData.users[senderId - 1].userName}
+            </UserName>
+            <Bubble color="#fff" flexDirection="row">
+              {text}
+            </Bubble>
+          </div>
+          <TimeStyle>{time}</TimeStyle>
+        </BubbleList>
+      )}
+    </>
+  );
+}
+
+const BubbleList = styled.div<{ justifyContent?: string }>`
+  display: flex;
+  justify-content: ${(props) => props.justifyContent || "flex-start"};
+`;
```

Comment:
```suggestion
const BubbleList = styled.div<{ justifyContent?: string }>`
  display: flex;
  justify-content: ${({ justifyContent }) => justifyContent || "flex-start"};
`;
```

`destructuring` 사용하시면 더 편리할 것 같아요! 
그리고 추가적인 제안사항인데요, 통일성을 위해서 `currentUser[0] === senderId`가 아닌 BubbleList에도 `justifyContent`를 `flex-start`로 부여해주고 

```css
justify-content: ${({ justifyContent }) => justifyContent};
```
이렇게 선언해도 좋을 것 같아요~